paasta_tools/cli/cmds/metastatus.py

```diff
@@ -64,6 +64,13 @@ def add_subparser(subparsers):
         help="define a different soa config directory",
     )
     status_parser.add_argument(
+        '-a', '--autoscaling-info',
```

Comment:
any reason to make this an optional thing? I don't know why we'd ever want to not see it.